code/modules/mob/living/carbon/life.dm

```diff
@@ -578,9 +578,9 @@ GLOBAL_LIST_INIT(ballmer_windows_me_msg, list("Yo man, what if, we like, uh, put
 /////////
 
 /mob/living/carbon/proc/handle_liver()
-	var/obj/item/organ/liver/liver = getorganslot(ORGAN_SLOT_LIVER)
-	if((!dna && !liver) || (NOLIVER in dna.species.species_traits))
+	if(!dna || (NOLIVER in dna.species?.species_traits))

```

Comment:
If this was runtiming because of the lack of species I'm not sure if it will enter this if block correctly when species is missing

Reply:
What do you mean exactly? It worked fine when I tested it.

Reply:
I am 90% sure this is functionally equivalent to `(!dna || (dna.species && (NOLIVER in dna.species.species_traits)))

Reply:
Yeah, it is, just confirmed with some coderbus help and more thorough doc reading